### PR TITLE
Fix mis-typing in CLI command documentations

### PR DESCRIPTION
**Pull request by posquit0**
I found some mis-typed documentations in CLI.

### CODE REVIEW

**Comment by posquit0** [comment]
@rdrgmnzs I fixed. Thanks to let me know!

**Comment by mikesplain** [comment]
Hi @posquit0 do you mind squashing all this into a single commit? We don't have the ability to do that on merge so we prefer all related content in a given commit.  Thanks!

/ok-to-test

**Comment by posquit0** [comment]
Sorry. I squashed all commits to a single one. 👍

File: cmd/kops/validate.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ var (
 
 	1. All k8s masters are running and have "Ready" status.
 	2. All k8s nodes are running and have "Ready" status.
-	3. Componentstatues returns healthy for all components.
+	3. Component statues returns healthy for all components.
```
Comment (justinsb) [inline cmd/kops/validate.go]: I think this is a typo in the original (as well as the missing space) - it should be "Component status returns healthy..."

Comment (posquit0) [reply cmd/kops/validate.go]: Fixed again. good!

**Comment by posquit0** [comment]
Sorry, my editor is configured to strip all trailing spaces on saving file. I restored trailing spaces.
Thanks!

**Comment by posquit0** [comment]
Please check this again :)

**Comment by justinsb** [comment]
Wonderful - thank you @posquit0 

/approve
/lgtm

**Pull request merged**